persistence/src/main/scala/com/lightbend/lagom/javadsl/persistence/AggregateEventTag.scala

```diff
@@ -23,6 +22,101 @@ object AggregateEventTag {
    */
   def of[Event <: AggregateEvent[Event]](eventType: Class[Event], tag: String): AggregateEventTag[Event] =
     new AggregateEventTag(eventType, tag)
+
+  /**
+   * Create a sharded aggregate event tag.
+   *
+   * This is a convenience function that uses the name of the class as the tag name. Note that if the class name
+   * changes, the tag name must be retained, and so this method will no longer be suitable for use.
```

Comment:
I personally really don't like this "side-effect". I'd actually remove this method, and force users to explicitly specify a stable tag name. Also, I'd deprecate the `of` method that has a similar behavior.